OpenRA.Mods.Common/Commands/ChatCommands.cs

```diff
@@ -48,7 +48,10 @@ public bool OnChat(string playername, string message)
 
 		public void RegisterCommand(string name, IChatCommand command)
 		{
-			Commands.Add(name.ToLowerInvariant(), command);
+			name = name.ToLowerInvariant();
```

Comment:
Wouldnt it make more sense for the later command to take priority?  You could then simplify it down to one line Commands[name] = ...